cli/hai.py

```diff
@@ -76,8 +77,8 @@ async def send_to_hai(report, verbose):
     if verbose:
         end_time = time.time()
         execution_time = end_time - start_time
-        print(f"{bcolors.OKCYAN}Execution Time: {execution_time} seconds{bcolors.ENDC}")
-        print(f"{bcolors.OKBLUE}Responses from Hai: {bcolors.ENDC}")
+        print(colored(f"Execution Time: {execution_time} seconds", 'cyan'))
```

Comment:
```suggestion
        print(colored("Execution Time: {execution_time} seconds", 'cyan'))
```

Reply:
I suppose this was a mistake?

Reply:
No they are f-strings: https://docs.python.org/3/reference/lexical_analysis.html#f-strings